bc_obps/Makefile

```diff
@@ -50,6 +50,7 @@ create_db:
 	@$(PSQL) -d postgres -tc "SELECT count(*) FROM pg_database WHERE datname = '$(DB_NAME)'" | \
 		grep -q 1 || \
 		$(PSQL) -d postgres -c "CREATE DATABASE $(DB_NAME)";
+		$(PSQL) -d postgres -c "CREATE EXTENSION postgis";
```

Comment:
Let's avoid postgis for now, we don't have a use case for it and it complicates our environment setup